### PR TITLE
docs: added note on github maven permissions

### PR DESCRIPTION
**Pull request by babadofar**
# Description
Added some info on the necessary permissions for using github as maven

### CODE REVIEW

File: content/en/v3/admin/setup/config/repository.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ repositories:
     snapshotURL: https://maven.pkg.github.com/myowner/mysnapshots/
 ```
 
+Note: The pipeline user will be used to push maven packages, so you may have to update the token for the pipeline user with additional permissions.
```
Comment (ankitm123) [inline content/en/v3/admin/setup/config/repository.md:93]: Can we add which additional permissions need to be added?

Comment (babadofar) [reply content/en/v3/admin/setup/config/repository.md:93]: sure

Comment (ankitm123) [reply content/en/v3/admin/setup/config/repository.md:93]: or link any documentation from github? that would help the users ...

**Pull request merged**